reaktive-test/build.gradle

```diff
@@ -0,0 +1,53 @@
+setupMultiplatformLibrary(project)
+
+kotlin {
+    sourceSets {
+        commonMain {
+            dependencies {
+                implementation project(':reaktive')
+            }
+        }
+    }
+}
+
+final target = currentTarget()
+
+if (target.isCommon() || target.isMeta()) {
+    setupCommonTargets(project)
+
+    kotlin {
+        sourceSets {
+            jvmMain.dependsOn commonMain
+            jvmTest.dependsOn commonTest
+            
+            androidMain.dependsOn commonMain
```

Comment:
Do we need android for this project? Just publish jvm version.

Reply:
I'm not sure, I don't see any downsides of this. I wanted to keep consistency. Also later we might add some android specific code and it might be a braking change perhaps. Not sure, what do you guys think?